Source/WebCore/rendering/RenderListBox.cpp

```diff
@@ -498,7 +501,10 @@ void RenderListBox::paintItemBackground(PaintInfo& paintInfo, const LayoutPoint&
 {
     const auto& listItems = selectElement().listItems();
     RefPtr listItemElement = listItems[listIndex].get();
-    auto& itemStyle = *listItemElement->computedStyle();
+    auto computedRenderStyle = listItemElement->computedStyle();
```

Comment:
Just keep the `itemStyle` name and use `itemStyle->` for the two usages later in the function.